src/components/Home/Aditya/Week4/pages/Week3/utils/functions.ts

```diff
@@ -0,0 +1,11 @@
+export const removeWhitespace = (str: string): string => {
+  let currentTask = str.replace(/\s+/g, ' ');
+  if (currentTask[0] === ' ') {
+    currentTask = currentTask.slice(1);
+  }
+  return currentTask;
+};
+
+export const getTodayDate = (): string => {
+  return new Date().toISOString().slice(0, 10);
```

Comment:
what is the difference between this file and `src/components/Home/Aditya/Week4/pages/Week1/utils/utils.ts`